cmd/lambda/accounts/get.go

```diff
@@ -20,7 +22,8 @@ func GetAllAccounts(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		errorMessage := fmt.Sprintf("Failed to query database: %s", err)
 		log.Print(errorMessage)
-		response.WriteServerErrorWithResponse(w, errorMessage)
+		api.WriteAPIErrorResponse(w, dceerr.NewInternalServer(errorMessage, nil))
```

Comment:
This may not feel that different but when we move more logic away from this layer we can just raise errors and have them given to the customer appropriately.   This is a middle step showing how it will work but this code will get more simple with the next set of work.